config.py

```diff
@@ -88,7 +88,7 @@ class DevelopmentConfig(Config):
 
     ACCOUNT_SERVICE_URL = os.getenv("ACCOUNT_SERVICE_URL", "http://localhost:8082/surveys/todo")
     ACCOUNT_SERVICE_LOG_OUT_URL = os.getenv("ACCOUNT_SERVICE_LOG_OUT_URL", "http://localhost:8082/sign-in/logout")
-    EQ_URL = os.getenv("EQ_URL", "https://localhost:8086/session?token=")
```

Comment:
why change to http?

Reply:
When I ran frontstage locally it'd fail to get to mock-eq as it doesn't have a certificate so it kept failing.  Changing it to http fixes it, and all the others in this file are http which means that's probably the right move too